File: DailyCode/Classes/Program2.java
class Demo{
           int x=10;
	   int y=20;
	   void fun(){
	   
	   System.out.println("In fun");
	 
	   }
	       
	   void run(){

           System.out.println("In run");

           }  
	   public static void main(String[] args){
	  Demo obj = new Demo();
	  
	  System.out.println(obj.x);    //10
	  
	  System.out.println(obj.y);   //20
	 
          obj.fun();                   //In fun
	  obj.run();                   //In run
	   }


}

File: DailyCode/Package/35/OverLoding/Program2.cpp

// Function overloding - name mangling

#include<iostream>
class Demo {
	public:       
	void add(int x, int y){ //_Z3addii
		       std::cout << x+y << std::endl;


	       } 
              void add(int x, int y,int z){ //_Z3addiii
              std::cout << x+y+z << std::endl;

	   }
};
int main(){
              Demo obj;
              obj.add(20,30);
              obj.add(20,30,40);
}


